ptrack-gentest/cleanCloseObjectCluster.m

```diff
@@ -0,0 +1,57 @@
+function [xcheck,ycheck,Nfinal]=cleanCloseObjectCluster(x,y, cx, cy, D, L, M)

```

Comment:
Function is missing comments, input/output data types and format.

Reply:
documentation added, no data types as matlab/octave does no requires data types

Reply:
Documentation shouldn't be at the top of the function?

Reply:
no, it is matlab style docs